modules/linear_boltzmann_solvers/executors/nl_keigen.cc

```diff
@@ -2,55 +2,61 @@
 // SPDX-License-Identifier: MIT
 
 #include "modules/linear_boltzmann_solvers/executors/nl_keigen.h"
-
+#include "modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/power_iteration_keigen.h"
 #include "framework/object_factory.h"
 #include "framework/logging/log.h"
 
-#include "modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/power_iteration_keigen.h"
-
 namespace opensn
 {
 namespace lbs
 {
 
-OpenSnRegisterObjectInNamespace(lbs, XXNonLinearKEigen);
+OpenSnRegisterObjectInNamespace(lbs, NonLinearKEigen);
 
 InputParameters
-XXNonLinearKEigen::GetInputParameters()
+NonLinearKEigen::GetInputParameters()
 {
   InputParameters params = opensn::Solver::GetInputParameters();
 
-  params.SetGeneralDescription("Generalized implementation of a non-linear k-Eigenvalue solver");
+  params.SetGeneralDescription("Implementation of a non-linear k-Eigenvalue solver");
   params.SetDocGroup("LBSExecutors");
 
-  params.ChangeExistingParamToOptional("name", "XXPowerIterationKEigen");
+  params.ChangeExistingParamToOptional("name", "PowerIterationKEigen");
 
   params.AddRequiredParameter<size_t>("lbs_solver_handle", "Handle to an existing lbs solver");
 
   params.AddOptionalParameter("nl_abs_tol", 1.0e-8, "Non-linear absolute tolerance");
+
   params.AddOptionalParameter("nl_rel_tol", 1.0e-8, "Non-linear relative tolerance");
+
   params.AddOptionalParameter("nl_sol_tol", 1.0e-50, "Non-linear solution tolerance");
+
   params.AddOptionalParameter("nl_max_its", 50, "Non-linear maximum iterations");
 
   params.AddOptionalParameter("l_abs_tol", 1.0e-8, "Linear absolute tolerance");
+
   params.AddOptionalParameter("l_rel_tol", 1.0e-8, "Linear relative tolerance");
+
   params.AddOptionalParameter("l_div_tol", 1.0e6, "Linear divergence tolerance");
+
   params.AddOptionalParameter("l_max_its", 50, "Linear maximum iterations");
+
   params.AddOptionalParameter("l_gmres_restart_intvl", 30, "GMRes restart interval");
+
```

Comment:
I personally like the more compact version with no empty lines between parameters. How do others see this?

Reply:
I'm good either way on this. Happy to go with the more compact form.

Reply:
@wdhawkins : I agree with @andrsd

Reply:
I agree with @andrsd as well.

Reply:
Fixed.